scripts/globals/roe_records.lua

```diff
@@ -5868,6 +5953,336 @@ function getRoeRecords(triggers)
             reward = { sparks = 300, xp = 1500, accolades = 30 },
         },
 
+        -----------------------------------
+        -- Content (Limbus)
+        -----------------------------------
+
+        [772] =
+        { -- Spoils - Ivory Chip
+            trigger = triggers.itemLooted,
+            goal = 1,
+            reqs = { itemID = set { 1904 } },
+            flags = set { "repeat" },
+            reward = { sparks = 300, xp = 1500, accolades = 30 },
+        },
+
+        [773] =
+        { -- Spoils - Scarlet Chip
+            trigger = triggers.itemLooted,
+            goal = 1,
+            reqs = { itemID = set { 1905 } },
+            flags = set { "repeat" },
+            reward = { sparks = 300, xp = 1500, accolades = 30 },
+        },
+
+        [774] =
+        { -- Spoils - Emerald Chip
+            trigger = triggers.itemLooted,
+            goal = 1,
+            reqs = { itemID = set { 1906 } },
+            flags = set { "repeat" },
+            reward = { sparks = 300, xp = 1500, accolades = 30 },
+        },
+
+        [775] =
+        { -- Spoils - Orchid Chip
+            trigger = triggers.itemLooted,
+            goal = 1,
+            reqs = { itemID = set { 1986 } },
+            flags = set { "repeat" },
+            reward = { sparks = 300, xp = 1500, accolades = 30 },
+        },
+
+        [776] =
+        { -- Spoils - Cerulean Chip
+            trigger = triggers.itemLooted,
+            goal = 1,
+            reqs = { itemID = set { 1908 } },
+            flags = set { "repeat" },
+            reward = { sparks = 300, xp = 1500, accolades = 30 },
+        },
+
+        [777] =
+        { -- Spoils - Silver Chip
+            trigger = triggers.itemLooted,
+            goal = 1,
+            reqs = { itemID = set { 1907 } },
+            flags = set { "repeat" },
+            reward = { sparks = 300, xp = 1500, accolades = 30 },
+        },
+
+        [778] =
+        { -- Spoils - Smoky Chip
+            trigger = triggers.itemLooted,
+            goal = 1,
+            reqs = { itemID = set { 1910 } },
```

Comment:
This raw item IDs could be converted to the item.lua file.
Some of them may already be in there. I which case, we should be using the item reference.

Raw IDs = bad

Reply:
To document: Some chatter has occurred in DMs and this will be done, along with all existing itemIDs. In addition, mob kill triggers will be moved to the script files for the mobs themselves.